examples/hacker-news/esy.json

```diff
@@ -34,6 +35,7 @@
     "@opam/httpkit": "ostera/httpkit:httpkit.opam#5fd8ed4",
     "@opam/httpaf": "anmonteiro/httpaf:httpaf.opam#57e9dd2",
     "@opam/httpaf-lwt": "anmonteiro/httpaf:httpaf-lwt.opam#57e9dd2",
-    "@opam/zarith": "briskml/zarith#3fe8bfc"
+    "@opam/zarith": "briskml/zarith#3fe8bfc",
+    "flex": "briskml/flex#7042e49"
```

Comment:
We don't need a resolution for flex as we are pinning the dep directly to the commit.